src/api/v1/db/oracledb/connection.js

```diff
@@ -11,7 +11,7 @@ oracledb.outFormat = oracledb.OBJECT;
 oracledb.fetchAsString = [oracledb.DATE, oracledb.NUMBER];
 
 /** Increase 1 extra thread for every 5 pools but no more than 128 */
-const threadPoolSize = dbConfig.poolMax + (dbConfig.poolMax / 5);
+const threadPoolSize = dbConfig.poolMax + dbConfig.poolMax / 5;
```

Comment:
I was using prettier for formatting at one point and it automatically made the change. I don't have a preference either way and I think it would actually make sense to change it back to how you originally wrote it. I think you're right that the parentheses are probably more readable.